cmd/generate_resources.go

```diff
@@ -337,63 +337,42 @@ func getCurrencyMappingForLocale(locale string) CurrencyMapping {
 	// Normalize locale to lowercase for comparison
 	localeKey := strings.ToLower(locale)
 
-	// Check if locale is fr_CA (special case - treated as French but with different USD/AUD format)
-	if localeKey == "fr_ca" {
+	// Check if locale is GB-related (United Kingdom), such as en_GB
+	if strings.HasSuffix(localeKey, "_gb") {
 		return CurrencyMapping{
-			CAD: "$",   // CAD -> $
-			USD: "$US", // USD -> $US
-			AUD: "$AU", // AUD -> $AU
-			GBP: "£",   // GBP -> £
-			EUR: "€",   // EUR -> €
-		}
-	}
-
-	// Check if locale is French-related (but not fr_CA)
-	if strings.HasPrefix(localeKey, "fr") {
-		return CurrencyMapping{
-			CAD: "$CA", // CAD -> $CA
-			USD: "$US", // USD -> $US
-			AUD: "$AU", // AUD -> $AU
-			GBP: "£",   // GBP -> £
-			EUR: "€",   // EUR -> €
+			GBP: "£", // GBP -> £
+			EUR: "€", // EUR -> €
 		}
 	}
 
 	// Check if locale is CA-related (Canada), such as en_CA
 	if strings.HasSuffix(localeKey, "_ca") {
 		return CurrencyMapping{
-			CAD: "$",   // CAD -> $
-			USD: "US$", // USD -> US$
-			AUD: "AU$", // AUD -> AU$
-			GBP: "£",   // GBP -> £
-			EUR: "€",   // EUR -> €
+			CAD: "$", // CAD -> $
+			EUR: "€", // EUR -> €
 		}
 	}
 
 	// Check if locale is AU-related (Australia)
 	if strings.HasSuffix(localeKey, "_au") {
 		return CurrencyMapping{
-			CAD: "CA$", // CAD -> CA$
-			USD: "US$", // USD -> US$
-			AUD: "$",   // AUD -> $
-			GBP: "£",   // GBP -> £
-			EUR: "€",   // EUR -> €
+			AUD: "$", // AUD -> $
+			EUR: "€", // EUR -> €
 		}
```

Comment:
Do we need these two? AUD is already $ in _au locales and CAD is already $ in _ca locales, isn't it? (cf https://github.com/go-playground/locales/tree/master/en_AU/en_AU.go and https://github.com/go-playground/locales/blob/master/fr_CA/fr_CA.go)

Reply:
Ah, you are right, I have removed these bits from the script now